pkg/render/policyrecommendation.go

```diff
@@ -163,15 +163,14 @@ func (pr *policyRecommendationComponent) clusterRole() client.Object {
 	}
 }
 
-func (pr *policyRecommendationComponent) roleBinding() client.Object {
-	return &rbacv1.RoleBinding{
+func (pr *policyRecommendationComponent) clusterRoleBinding() client.Object {
```

Comment:
Does this really need to be a cluster role binding? 

RoleBindings (namespaced) can reference ClusterRoles. The reason to make it a cluster role binding is to bind it to multiple service accounts across multiple namespaces, which I don't think we're doing?